azure/code/rapids_csp_azure.py

```diff
@@ -181,6 +185,15 @@ def load_data(self, filename = 'dataset.orc', col_labels = None, y_label = 'ArrD
                         dataset = pyarrow_orc.ORCFile(file).read().to_pandas()
                 elif 'CSV' in self.data_type:
                     dataset = pd.read_csv( target_filename, names = col_labels )
+                    
+                elif 'Parquet' in self.data_type:
```

Comment:
This type of checking can be dangerous, and is probably not doing exactly what is expected. Its checking whether or not Parquet is contained within the self.data_type string, not that they are equal. If you want to test for equality using 'in', you probably should do something like:

'parquet' in (self.data_type,), or 'parquet' == self.data_type

Reply:
** Same comment for all the other 'in' tests.

Reply:
We had kept it consistent with aws and looks like data_type is the only test that's checking for equality. Using `==` operator for model_types, compute_type is checking whether it's contained in the string since it checks for single vs multi and CPU vs GPU. Thanks for reviewing!